surveys/urls.py

```diff
@@ -6,13 +6,13 @@
 urlpatterns = [
     url(r'^forms/(?P<pk>\d+)/?$', views_api.SurveyView.as_view(), name='survey-form'),
     url(
-        r'^forms/(?P<pk>\d+)/respond/?$',
-        views_api.SurveyPostView.as_view(),
-        name='survey-post',
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>[^/]+)/?$',
```

Comment:
Yes - seems dangerous to me - not sure what happens if you try and give it strange charsets!